src/libraries/Microsoft.PowerFx.Core/Public/Types/RecordType.cs

```diff
@@ -96,5 +97,30 @@ public static RecordType Empty()
         {
             return new KnownRecordType();
         }
+
+        internal override string DefaultExpressionValue()
```

Comment:
>DefaultExpressionValue [](http://example.com/codeflow?start=33&length=22)

This could be

  internal override void DefaultExpressionValue(StringBuilder sb);

that would compose more efficiently